### ISSUE TITLE
Make `@putoutjs/printer` options available to putout

### ISSUE THREAD

**Issue by magne4000** [coderaiser/putout#144]
Is it possible to have [`@putoutjs/printer` options](https://github.com/putoutjs/printer#overrides) available in `putout` API? For example:

```ts
import putout from "putout";

export function cleanImports(code: string): string {
  const result = putout(code, {
    isTS: true,
    printer: "putout",
    processors: ["typescript"],
    plugins: ["remove-unused-variables"],
    // Like this
    printerOptions: {
      format: {
        indent: "  ",
      },
    },
  });
  return result.code;
}
```

**Comment by coderaiser** [comment]
Landed support in [`@putout/engine-parser@6.1.0`](https://github.com/coderaiser/putout/tree/master/packages/engine-parser#api).

Re-install 🐊**Putout** and use:

```js
import putout from "putout";

export function cleanImports(code: string): string {
  const result = putout(code, {
    isTS: true,
    printer: ["putout", {
        format: {
            indent: "  ",
        },
    }],
    processors: ["typescript"],
    plugins: ["remove-unused-variables"],
  });
  
  return result.code;
```

Is it works for you?

**Comment by magne4000** [comment]
@coderaiser neat! I works like a charm :ok_hand: thanks
